### DISCUSSION TITLE
Finding the docket from opinion hit

### DISCUSSION THREAD

**Discussion by sliedes** [freelawproject/courtlistener#4414] [General]
Hi,

I often have trouble finding a docket of a case even given access to an opinion, and I wonder whether 

1. I'm doing something wrong
2. It may be that the docket is not on courtlistener even if the opinion may be
3. It's an inherently hard problem.

Background: I'm just a hobbyist/nerd following some themes, often coinciding with things like what EFF advocates for. I like reading opinions that people often post on the Internet, but often would like to delve deeper.

I'll walk you step by step where I'm coming from as a use case:

1. I read an EFF article about a 5th Circuit Court of Appeals decision: https://www.eff.org/deeplinks/2024/08/federal-appeals-court-finds-geofence-warrants-are-categorically-unconstitutional
2. That page links to the opinion: https://www.ca5.uscourts.gov/opinions/pub/23/23-60321-CR0.pdf
3. Reading that, I decide that I'd like to take a look at the other documents of the case (or, in this case, because some EFF post mentioned "heading to SCOTUS", to see if there's a notice of appeal docketed). For that, I need to find the docket. For me, the most ideal option would be some tool to get directly from this case to a docket. Not having that, I note the details: Case number **23-60321**, **5th Circuit**, **US v Smith**. I usually try first Google by putting in those and adding "docket", but in this case it doesn't find the right stuff, and often it leads to some subscription services or only finds that decision.
4. I go to courtlistener.com and enter "23-60321 smith" in the search box. It finds this opinion, but not the docket. The title looks like a case name, but takes me to the document, not the docket.

![image](https://github.com/user-attachments/assets/904fc032-7659-4890-b21e-aa07b3d28c8f)

5. When I go to the document page, it contains the OCR'd text, but to the best of my ability to find there's no link to the docket. I realize it may be that the docket has not been RECAP'd, but I don't know if this is the case.

---

Ideally, I think the easiest UX would be if there was just a way to directly go from a PDF to a docket (sounds like some browser extension) or to learn that it does not exist in RECAP/courtlistener. The second best option would be some neat procedure to get from these case details to that same information.

Any thoughts?

**Comment by mlissner** [comment]
Sure, so two things:

1. You should search RECAP for federal cases. We're working on our UX to make this more obvious, but until then, use https://www.courtlistener.com/recap/

2. If you have a storage.courtlistener.com link, you can hack at it to load the docket:

![recap-url-hack](https://github.com/user-attachments/assets/08bf482a-d40b-45e7-914a-481e4c69e128)
